index.js

```diff
@@ -1,13 +1,38 @@
+// Use UTC for for all Date parsing
+process.env.TZ = 'UTC';
```

Comment:
that's sooo helpful.

Reply:
We also need to figure out what we want to do about timezones. Right now all times show up as UTC. This is mostly an issue for relative times like `tomorrow morning`.

GitHub.com uses frontend JS to get the user's timezone offset, but there's not a way for us to do that right now per-user.

Short term I was thinking we could add `timezone` to the config file as the default for the repo.